datagrepper/app.py

```diff
@@ -281,7 +249,8 @@ def raw():
     start = flask.request.args.get("start", None)
     end = flask.request.args.get("end", None)
     delta = flask.request.args.get("delta")
-    start, end, delta = assemble_timerange(start, end, delta)
+    with app.app_context():
```

Comment:
This shouldn't be here, the app context is already created when the app starts. If your unit tests fails because the app context is not created, it should be done in the unit test itself.